CHANGELOG.md

```diff
@@ -23,6 +23,10 @@
   Fixed a bug introduced in #4047 that limited the `--level` option of logger
   to Pascal-cased values (e.g. accepting "Info", but not "info"). It now
   ignores case again.
+- [#4286](https://github.com/firecracker-microvm/firecracker/pull/4286):
```

Comment:
can reference the backport PR

Reply:
Thanks for catching this. Updated!

Reply:
worth changing the second occurrence as well :)

Reply:
Well done me 🤡

Reply:
Ok done, along with a fix for the dependencies check.